packages/thirdweb/src/rpc/rpc.ts

```diff
@@ -138,46 +138,41 @@ export function getRpcClient(
         requestTimeoutMs: options.config?.requestTimeoutMs,
       })
         .then((responses) => {
-          // for each response, resolve the inflight request
           activeBatch.forEach((inflight, index) => {
+            // Handle the inflight request promise for each response.
```

Comment:
These changes are just comments and re-arranging the if statements into if-else's.